satellite/audit/integration_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"storj.io/storj/internal/memory"
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/internal/testplanet"
+	"storj.io/storj/internal/testrand"
+	"storj.io/storj/pkg/storj"
+)
+
+func TestChoreAndWorkerIntegration(t *testing.T) {
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount: 1, StorageNodeCount: 5, UplinkCount: 1,
+	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+		satellite := planet.Satellites[0]
+		satellite.Audit.Worker.Loop.Pause()
+		satellite.Audit.Chore.Loop.Pause()
+
+		ul := planet.Uplinks[0]
+
+		// Upload 2 remote files with 1 segment.
+		for i := 0; i < 2; i++ {
+			testData := testrand.Bytes(8 * memory.KiB)
+			path := "/some/remote/path/" + string(i)
```

Comment:
nit:
`string(i)` translates to the UTF-8 character indexed by the `i` rather than string representation of its value. While it isn't a problem if at some point we want to print those paths for debugging or whatever else, it will be harder to read them

https://play.golang.org/p/1XIzSk8NSYH

I would prefer to use something like `path := "/some/remote/path/" + strconv.Itoa(i)`